examples/istio-based-service-mesh/observability/kiali/azure-managed-prometheus.md

```diff
@@ -0,0 +1,290 @@
+# Kiali with Azure Monitor Managed Prometheus
+
+This guide shows how to set up Kiali dashboard with Azure Monitor managed Prometheus for Istio service mesh visualization on AKS.
+
+> [!Important]
+> Complete the [prerequisites and setup](README.md) before following this guide.
+
+## Set additional environment variables
```

Comment:
I think this set additional env variables can come later after step 1. after creating the monitor workspace.